CallKitDemo/SpeakerboxCallManager.swift

```diff
@@ -11,6 +11,12 @@ import CallKit
 import OpenTok
 
 final class SpeakerboxCallManager: NSObject {
+    
+    enum CallAction: String {
```

Comment:
How about calling this `Call` instead of `CallAction` ?

Reply:
Updated!